salt/metalk8s/orchestrate/downgrade/precheck.sls

```diff
@@ -4,7 +4,7 @@
 {%- set nodes_versions = pillar.metalk8s.nodes.values() | map(attribute='version') | list %}
 {%- do nodes_versions.sort(cmp=salt.pkg.version_cmp, reverse=True) %}
 {%- set expected = nodes_versions | first %}
-{%- if saltenv != 'metalk8s-' ~  expected %}
+{%- if salt.pkg.version_cmp(saltenv, 'metalk8s-' ~ expected) >= 0 %}
```

Comment:
`version_cmp` on stuff with 'metalk8s-' as a prefix? Wouldn't it make more sense with:

```suggestion
{%- if salt.pkg.version_cmp(saltenv | replace('metalk8s-', ''), expected) >= 0 %}
```

or maybe `saltenv[9:]`, but that's less explicit.

Reply:
Does not really change much but ok let's compare just version